zipline/data/data_portal.py

```diff
@@ -637,7 +638,51 @@ def _check_extra_sources(self, asset, column, dt):
 
                 raise KeyError
 
-    def get_spot_value(self, asset, field, dt, data_frequency):
+    def _get_overnight_adjustments(self, assets, field, dt):
+
+        if not isinstance(assets, (list, tuple)):
+            assets = [assets]
+
+        # dt at bts is forced back to 4 pm previous day for the purpose of
+        # getting prices, so we want to find adjustments that occur on the
+        # upcoming midnight
+        adjustment_dt = normalize_date(dt + timedelta(days=1))
+
+        adjs = []
+
+        split_adj_factor = lambda x: x if field != 'volume' else 1.0/x
+
+        for asset in assets:
+
+            split_adj = self._get_adjustment_list(
+                asset, self._splits_dict, "SPLITS"
+            )
+            split_adj = \
+                [split_adj_factor(adj) for (adj_dt, adj) in split_adj
+                 if adj_dt == adjustment_dt]
+
+            if field != 'volume':
+                merger_adj = self._get_adjustment_list(
+                    asset, self._mergers_dict, "MERGERS"
+                )
+                merger_adj = [adj for (adj, adj_dt) in merger_adj
+                              if adj_dt == adjustment_dt]
+
+                div_adj = self._get_adjustment_list(
+                    asset, self._dividends_dict, "DIVIDENDS",
+                )
+                div_adj = [adj for (adj, adj_dt) in div_adj
+                           if adj_dt == adjustment_dt]
+            else:
+                merger_adj = div_adj = []
+
+            ratio = reduce(mul, split_adj + merger_adj + div_adj, 1.0)
+            adjs.append(ratio)
+
+        return adjs
+
+    def get_spot_value(self, asset, field, dt, data_frequency,
+                       handle_bts=False):
```

Comment:
I don't think this should have a default value.  Providing one is inviting us to create bugs by accidentally passing the wrong value.

As a general rule of thumb, business logic functions that aren't called by users should almost never use default values.